### PR TITLE
Remove '\' from the chatbot output, if any

### PR DESCRIPTION
**Pull request by leungmanhin**

### CODE REVIEW

File: opencog/nlp/chatbot-psi/actions.scm [hunk 1]
```diff
@@ -72,8 +72,9 @@
 (define (say . words)
     (define utterance (string-join (map cog-name words)))
 
-    ; Remove those '[' and ']' that may exist in the output
-    (set! utterance (string-filter (lambda (c) (not (or (char=? #\[ c) (char=? #\] c)))) utterance))
+    ; Remove those '[', ']' and '\' that may exist in the output
```
Comment (linas) [inline opencog/nlp/chatbot-psi/actions.scm]: any clue where these are coming from?

Comment (leungmanhin) [inline opencog/nlp/chatbot-psi/actions.scm]: I hit one reply that has a WordNode `(WordNode "\"hello\".")`

I guess '\' may be used to escape quotes exist in some word in some aiml rule, and after converting to opencog psi rules they get the chance to be part of a word

Comment (linas) [inline opencog/nlp/chatbot-psi/actions.scm]: ooofdah. quotes are hard. Can you add a short comment that says `; The backslash is sometimes generated by AIML rules. The square brackets are sometimes generated by ...`  ?  In the short run, stripping stuff out is OK but In the long run, there should not be garbage in the WordNode string ...

Comment (leungmanhin) [inline opencog/nlp/chatbot-psi/actions.scm]: sure, I'll do that

**Pull request merged**